typescript/db/src/repository/index.ts

```diff
@@ -124,6 +124,12 @@ export const repository: Repository = {
       ).label.create({ data: label });
       return createdLabel.id;
     },
+    addMany: async ({ labels, imageId }, user) => {
+      await checkUserAccessToImage({ where: { id: imageId }, user });
+      const prisma = await getPrismaClient();
+      await prisma.label.createMany({ data: labels });
```

Comment:
You create the `labels` without checking their individual `imageId`:
- Either force them to the same `imageId` from the args.
- Or remove `imageId` from the args and check the rights to all image ids (obviously group them by identical ones first to limit queries)

Reply:
Fixed